service/src/main/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersController.java

```diff
@@ -148,7 +173,13 @@ private Document getDocument(@RequestHeader("authorization") String authorizatio
                                  Map<String, Object> templateData) {
         DocmosisDocument document = docmosisService.generateDocmosisDocument(templateData, DocmosisTemplates.SDO);
 
-        return uploadDocumentService.uploadPDF(userId, authorization, document.getBytes(),
-            "draft-standard-directions-order.pdf");
+        return uploadDocumentService.uploadPDF(userId, authorization, document.getBytes(), document.getDocumentTitle());
+    }
+
+    private String documentFilename(OrderStatus status) {
+        if (status == OrderStatus.SEALED) {
+            return OrderStatus.SEALED.getDocumentTitle();
+        }
+        return OrderStatus.DRAFT.getDocumentTitle();
```

Comment:
Method can be removed and just get the document title from where the it is called

Reply:
silly me, thanks 👍